modules/lb_membership_calculator/lb_membership_calculator.info.yml

```diff
@@ -0,0 +1,10 @@
+name: 'LB Membership Calculator'
+type: module
+description: 'Helper module to show "Membership Calculator" block in Layout Builder.'
```

Comment:
Let's have a README.md with how to install this piece
https://github.com/YCloudYUSA/yusaopeny_memberships#installation

Remember, membership framework is not a default distribution compinent. Would be great @froboy to jump in with some ideas
We can't have membership calculator in a distribution by  default